### PR TITLE
unify loki url

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

**Comment by daixiang0** [comment]
@sh0rez @rfratto  any update?

**Comment by daixiang0** [comment]
@sh0rez @rfratto any update?

File: cmd/docker-driver/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Overview
 
 Docker logging driver plugins extends Docker's logging capabilities. You can use Loki Docker logging driver plugin to send
-Docker container logs directly to your Loki instance or [Grafana Cloud](https://grafana.com/loki).
+Docker container logs directly to your Loki instance or [Grafana Cloud](https://grafana.com/oss/loki).
```
Comment (pstibrany) [inline cmd/docker-driver/README.md]: https://grafana.com/loki already redirects to https://grafana.com/oss/loki and may redirect elsewhere in the future. I don't see a need to do this.

**Comment by cyriltovena** [comment]
Consensus is we want to keep it that way. Thank you @daixiang0 !

**Pull request closed without merging**